server/apps/immich/src/config/app.config.ts

```diff
@@ -9,7 +9,6 @@ export const immichAppConfig: ConfigModuleOptions = {
     DB_USERNAME: Joi.string().required(),
     DB_PASSWORD: Joi.string().required(),
     DB_DATABASE_NAME: Joi.string().required(),
-    UPLOAD_LOCATION: Joi.string().required(),
```

Comment:
This is the only concern I have. If we remove this, and the user forgot to put in a location for this value the app won't have the location to upload the file to, thus the core functionality won't work. So enforcing this to run this app is a must in my opinion. What is your thought?